plugins/continuous_dataset.cc

```diff
@@ -188,10 +188,9 @@ struct ContinuousDataset::Itl {
         switch (val.type()) {
         case Json::objectValue:
             for (auto it = val.begin(), end = val.end();  it != end;  ++it) {
-                ColumnName newPrefix
-                    = prefix == ColumnName()
-                    ? ColumnName(it.memberName())
-                    : ColumnName(prefix.toUtf8String() + "." + it.memberName());
+                ColumnName newPrefix = prefix == ColumnName()
```

Comment:
Indeed, you can simplify this whole block to ExpressionValue::parseJson(..., ...)